template/koa-rest-api/package.json

```diff
@@ -1,8 +1,13 @@
 {
   "dependencies": {
     "@koa/router": "^10.1.1",
+    "@opentelemetry/api": "^1.0.3",
+    "@opentelemetry/exporter-collector-grpc": "^0.25.0",
+    "@opentelemetry/instrumentation-aws-sdk": "^0.1.0",
+    "@opentelemetry/instrumentation-http": "^0.26.0",
+    "@opentelemetry/sdk-node": "^0.26.0",
```

Comment:
Do we need to be concerned about these v0s? Does OpenTelemetry promise some degree of stability?